rust/kernel/chrdev.rs

```diff
@@ -96,10 +152,11 @@ impl<const N: usize> Registration<{ N }> {
             if res != 0 {
                 return Err(Error::from_kernel_errno(res));
             }
+            const NONE: Option<Cdev> = None;
             this.inner = Some(RegistrationInner {
                 dev,
                 used: 0,
-                cdevs: [MaybeUninit::<bindings::cdev>::uninit(); N],
+                cdevs: [NONE; N],
```

Comment:
Does just putting `None` here not work?

Reply:
Unfortunately not... see [here](https://github.com/Rust-for-Linux/linux/pull/207#pullrequestreview-640624695)